content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md

```diff
@@ -439,7 +439,262 @@ The onboard magnetometer of the Nicla Voice can be used to determine the board's
+  int16_t x_acc_raw, y_acc_raw, z_acc_raw, x_gyr_raw, y_gyr_raw, z_gyr_raw;
+  float x_acc, y_acc, z_acc, x_gyr, y_gyr, z_gyr;
+
+  // Read operation status variable.
+  int status;
+
+  // Perform data read from the BMI270 sensor. The data read includes accelerometer and gyroscope data.
+  // The sensor's read function is called with 0x0C as the start address and 16 as the number of bytes to read.
+  // Collected data is placed into sensor_data array.
+  status = NDP.sensorBMI270Read(READ_START_ADDRESS, READ_BYTE_COUNT, &sensor_data[0]);
+
+  // Check the status of the read operation.
+  CHECK_STATUS(status);
+
+  // Parse the read sensor data. Data is in 16-bit format, where lower byte comes first (little endian).
+  // Data for each axis (X, Y, Z) of the accelerometer and gyroscope is extracted from the array.
+  x_acc_raw = (0x0000 | sensor_data[0] | sensor_data[1] << 8);
+  y_acc_raw = (0x0000 | sensor_data[2] | sensor_data[3] << 8);
+  z_acc_raw = (0x0000 | sensor_data[4] | sensor_data[5] << 8);
+  x_gyr_raw = (0x0000 | sensor_data[6] | sensor_data[7] << 8);
+  y_gyr_raw = (0x0000 | sensor_data[8] | sensor_data[9] << 8);
+  z_gyr_raw = (0x0000 | sensor_data[10] | sensor_data[11] << 8);
+
+  // Convert raw accelerometer data to acceleration expressed in meters per second squared. 
+  x_acc = x_acc_raw * ACCEL_SCALE_FACTOR;
+  y_acc = y_acc_raw * ACCEL_SCALE_FACTOR;
+  z_acc = z_acc_raw * ACCEL_SCALE_FACTOR;
+
+  // Convert raw gyroscope data to angular velocity expressed in degrees per second.
+  x_gyr = x_gyr_raw * GYRO_SCALE_FACTOR;
+  y_gyr = y_gyr_raw * GYRO_SCALE_FACTOR;
+  z_gyr = z_gyr_raw * GYRO_SCALE_FACTOR;
+  
+  // Print accelerometer data (expressed in meters per second squared). 
+  Serial.print("x_acc:");
+  Serial.print(x_acc);
+  Serial.print(",");
+  Serial.print("y_acc:");
+  Serial.print(y_acc);
+  Serial.print(",");
+  Serial.print("z_acc:");
+  Serial.println(z_acc);
+
+  // Print gyroscope data (expressed in degrees per second). 
+  Serial.print("x_gyr:");
+  Serial.print(x_gyr);
+  Serial.print(",");
+  Serial.print("y_gyr:");
+  Serial.print(y_gyr);
+  Serial.print(",");
+  Serial.print("z_gyr:");
+  Serial.println(z_gyr);
+
+  delay(1000);
+}
+```
+
+Here you can find a step-by-step explanation of the code:
+
+First, the necessary libraries are included: 
+
+- `NDP.h` and `BMI270_Init.h` for the Nicla Voice board's basic functions and the IMU control.
+- Macros are defined for checking the status of the IMU; these macros allow the sketch to detect and handle sensor errors. 
+
+Next, user functions `ledBlueOn()`, `ledGreenOn()`, and `ledRedBlink()` definition: 
+
+- These functions allow the onboard LEDs to flash specific colors to indicate different states: blue for a successful match, green for an event, and red to indicate an error.
+
+Next, in the `setup()` function:
+
+- The serial communication is initialized at a baud rate of 115200.
+- The Nicla Voice board is initialized, and the LDO is disabled.

```

Comment:
explain in a couple of words what is the LDO and what we want to disable it

Something like:

The nicla board is initialized, and the LDO (the power regulator responsible of putting the Nicla on power-saving modes) is disabled to avoid reading disconnections for the IMU.